lago-types/src/models/customer.rs

```diff
@@ -128,6 +128,7 @@ pub struct CustomerIntegration {
 
 /// Supported integration types.
 #[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
```

Comment:
To be future proof ?

```suggestion
#[serde(rename_all = "snake_case")]
```